cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -459,7 +538,8 @@ func init() {
 	createCmd.Flags().IntVar(&masters, "masters", 1, "the number of masters to create")
 	createCmd.Flags().StringVar(&clusterCpus, "cpus", "2.0", "the share of CPUs as fraction (each container/VM)")
 	createCmd.Flags().IntVar(&clusterMemory, "memory", 2048, "the limit on memory usage in MB (each container/VM)")
-	createCmd.Flags().IntVar(&clusterDiskSize, "disk", 6*1024, "the limit on disk size in MB (each VM)")
+	createCmd.Flags().IntVar(&clusterDiskSize, "disk", 6*1024, "default limit on disk size in MB (each VM)")
+	createCmd.Flags().StringSliceVar(&clusterDisks, "user-disk", []string{}, "list of disks to create for each VM in format: <mount_point1>:<size1>;<mount_point2>:<size2>")
```

Comment:
need to update syntax: `;` -> `:` (`;` is bash special character)

Reply:
Shouldn't that be a comma?

Reply:
Comma is parsed by Cobra and what is intended to be a single disk with couple partitions, turns into several disks with a single partition.

Reply:
yeah, I've updated it in the code, but forgot update help